securedrop/sass/_base.sass

```diff
@@ -157,7 +157,6 @@
 
         a
           color: #333
-          text-decoration: none
```

Comment:
Verified the **designation** class is only used in `securedrop/journalist_templates/_source_row.html` and the only anchor in the inner HTML is the link that is modified. There is no risk this change can have a side effect on the look of other links in the source or journalist interface. 
Note to self: this chunk of sass belongs to the journalist interface and should not be in _base.sass because it shares nothing with the source interface.